scripts/set_deployment_image_tag.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Generic script that sets the image+tag of a container in a deployment's pod spec in user namespaces
+# As is.  Will set deployment's "r-studio-server" container's image tag to "v1.3.2" in all user namespaces
+
+set -e
+
+DRY_RUN=true
+
+while true; do
+	read -p "Is this a DRY RUN? i.e. NO-OP [y/N]: " dry_run
+	case $dry_run in
+	    [y]*    ) DRY_RUN=true; break;;
+	    [N]*    ) DRY_RUN=false; break;;
+	    *       ) echo "Answer y to simulate the operation.  Answer N to perform the operation."; exit;;
+    esac
+done
+
+APP=rstudio
```

Comment:
If this is not a migration but a script will run often it may be worth having these 3 constants read from environment variables with a good default.